worlds/zillion/item.py

```diff
@@ -1,6 +1,34 @@
+from typing import Counter
 from BaseClasses import Item, ItemClassification as IC
 from zilliandomizer.logic_components.items import Item as ZzItem
 
+_useful_thresholds = {
+    "Apple": 9999,
+    "Champ": 9999,
+    "JJ": 9999,
+    "Win": 9999,
+    "Empty": 0,
+    "ID Card": 10,
+    "Red ID Card": 2,
+    "Floppy Disk": 7,
+    "Bread": 0,
+    "Opa-Opa": 20,
+    "Zillion": 8,
+    "Scope": 8,
+}
+""" make the item useful if the number in the item pool is below this number """
+
+
+def get_classification(name: str, zz_item: ZzItem, item_counts: Counter[str]) -> IC:
```

Comment:
I assume this not accounting for extra items such as from start_inventory, item_plando, and item_links is intended / expected?

Reply:
I admit I didn't think of those when writing this.

But thinking about it now, I think it's not worth the added complexity for something that won't make very much of a difference.

Reply:
I agree that it's not worth changing, especially since this'll only ever make items Useful when they shouldn't be, and not the reverse.